pkg/server/proto/server.proto

```diff
@@ -1087,6 +1105,32 @@ message TaskLaunchInfo {
   repeated string arguments = 3;
 }
 
+// TrackTask tracks the life of an on-demand runner task that spawns Start
+// and Stop tasks for any kind of job/operation in Waypoint. Automatic jobs
+// such as project polling or status report generation spawn on-demand
+// runner tasks, and this message can be used to track the life of those
+// automated jobs. Note that every on-demand runner task is wrapped with a
+// Start and Stop task, which we track here as well.
+message TrackTask {
```

Comment:
Would just rename to `Task` (along with everything else, see first review comment which explains why)

Reply:
Done!